src/components/Common/Forms.js

```diff
@@ -71,3 +82,102 @@ export const Fieldset = styled.section`
     margin-bottom: ${remcalc(24)};
   }
 `;
+
+const fadeIn = keyframes`
+  from{
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+`;
+
+export const SearchField = styled.section`
+  position: relative;
+  margin-bottom: ${remcalc(36)};
+`;
+
+export const Results = styled.ul`
+  width: 100%;
+  max-height: ${remcalc(200)};
+  list-style-type: none;
+  background: ${props => props.theme.colors.greyBg};
+  position: absolute;
+  left: 0;
+  top: ${remcalc(56)};
+  z-index: 10;
+  overflow-x: auto;
+  overflow-y: scroll;
+  will-change: opacity;
+  animation: ${fadeIn} ${props => props.theme.animations.long} linear;
+`;
+
+export const Result = styled.li`
+  padding: ${remcalc(8)} ${remcalc(24)};
+  font-size: ${remcalc(18)};
+  line-height: ${remcalc(24)};
+  font-weight: 300;
+  color: ${props => props.theme.colors.text};
+  background: transparent;
+  transition: background ${props => props.theme.animations.normal} ease-in-out,
+    color ${props => props.theme.animations.normal} ease-in-out;
+  &:hover {
+    background: ${props => props.theme.colors.vibrant};
+    > a {
+      color: ${props => props.theme.colors.blueBg};
+    }
+  }
+  ${is('notFound')`
+    pointer-events:none;
+    color: ${props => props.theme.colors.grey};
+    font-weight: 500;
+  `};
+`;
+
+export const SearchBox = forwardRef(
```

Comment:
This component and its styles is so big and advanced that it should def get its own file, not together with the simple form components.

Reply:
I just wanted to keep all form UI in the same place. But I agree with you